backend/fbdev.c

```diff
@@ -22,6 +22,25 @@
 #define SCREEN(x) ((twin_context_t *) x)->screen
 #define PRIV(x) ((twin_fbdev_t *) ((twin_context_t *) x)->priv)
 
+/* color conversion */
```

Comment:
Move the definitions of `ARGB32_TO_RGB565_PERLINE`, `ARGB32_TO_RGB888_PERLINE`, and `ARGB32_TO_ARGB32_PERLINE` near by `FBDEV_PUT_SPAN_IMPL`. i.e., somewhere after `twin_fbdev_t`.